packages/decision-tree/src/DecitionTree.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { DecisionTree } from './DecisionTree';
+import { renderWithProviders, screen, fireEvent, waitFor } from '@tests/renderWithProviders';
+import { questionnaire, serverMockup } from './tests/Questionnaire.mock';
+import { UseSetupCallbackCB } from '@types';
+
+describe('DecisionTree', () => {
+  const callback = async function (questionId, value) {
+    console.log(questionId, value);
+    return serverMockup[questionId];
+  } as UseSetupCallbackCB;
+
+  test('should render all the components', () => {
+    renderWithProviders(<DecisionTree questionnaire={questionnaire} callback={callback} />);
+    expect(screen.getAllByText('Next')).toHaveLength(1);
+    expect(screen.getAllByText('Back')).toHaveLength(1);
+    expect(screen.getAllByText('Male')).toHaveLength(1);
+    expect(screen.getAllByText('Female')).toHaveLength(1);
+    expect(screen.getAllByText('What sex were you assigned at birth?')).toHaveLength(1);
```

Comment:
we should use `getByText` instead of `getAllByText` where there is just one element